src/core/dash_internal.h

```diff
@@ -1506,46 +1506,22 @@ template <bool UV>
 std::enable_if_t<UV, unsigned> Segment<Key, Value, Policy>::CVCOnBump(uint64_t ver_threshold,
                                                                       unsigned bid, unsigned slot,
                                                                       Hash_t hash,
-                                                                      uint8_t result_bid[2]) const {
+                                                                      uint8_t result_bid[3]) const {
```

Comment:
The problem here now is that we might send more buckets than we actually changed because there exist fast paths in `Segment::BumpUp`. For example, if we got empty space for some reason we will just unload the stash entry leaving the rest of the buckets `untouched`. As this was broken before, this is a small price for now but we should improve. I am still not 100% familiar with the dash code but I will investigate how we can patch this and make it better